lib/paypal_express/setup.rb

```diff
@@ -76,15 +76,21 @@ def paypal_params
       end
 
       # @method shipping_methods
-      # 
+      #
       # @returns shipping methods with promotions applied
       def shipping_methods
         @shipping_methods ||= ShippingMethodsForCart.new(cart: cart, shipping_methods: shipping_method_model.all).call.sort_by(&:total)
       end
 
       def validate_shipping_method
-        unless cart.shipping_method_id.nil? || shipping_methods.any? {|sm| sm.id == cart.shipping_method_id} then
-          raise ::FlexCommerce::PaypalExpress::Exception::AccessDenied.new(I18n.t("payment_setup.shipping_method_not_available"))
```

Comment:
This translation wasn't loading and we were getting a translation missing error in the test runs. It was the only one in the locale file anyway, so I've moved it into the library. We could add another error class here as well (open to naming ideas)